ad5758-sdz/ad5758.c

```diff
@@ -775,7 +775,7 @@ int32_t ad5758_init(struct ad5758_dev **device,
 	ret |= gpio_set_value(dev->ldac_n, GPIO_LOW);
 
 	/* Initialize the SPI communication */
-	ret |= spi_init(&dev->spi_desc, &init_param.spi_init);
+	ret |= spi_init(&dev->spi_desc, init_param.spi_init);

```

Comment:
Although this fixes the compile error, I think a better approach is to update the `spi_init()` function from the platform_drivers